src/chainparams.cpp

```diff
@@ -921,6 +921,7 @@ class CRegTestParams : public CChainParams {
         UpdateLLMQTestParametersFromArgs(args, Consensus::LLMQType::LLMQ_TEST_INSTANTSEND);
         UpdateLLMQTestParametersFromArgs(args, Consensus::LLMQType::LLMQ_TEST_PLATFORM);
         UpdateLLMQInstantSendDIP0024FromArgs(args);
+        assert(consensus.V20Height >= consensus.DIP0003Height);
```

Comment:
why?

Reply:
because of conversation here: https://github.com/dashpay/dash/pull/6656/files#r2075111410